index.html

```diff
@@ -13,7 +13,18 @@
 
 <body>
     <div class="container">
-   
+        
+        <nav>
+            <img src="img/lambda-black.png" alt="Lambda Logo">
```

Comment:
It wouldn't hurt to also link your logo back to the home page. It's good for UX.